drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -73,7 +73,7 @@ static uint32_t get_pllsrc_frequency(size_t pll_id)
 
 static uint32_t get_startup_frequency(void)
 {
-	switch (LL_RCC_GetSysClkSource()) {
+	switch (LL_RCC_GetClkAfterWakeFromStop()) {
```

Comment:
We'd miss an explanation on why this function is providing correct result also at startup